pkg/granted/credentialsprocess.go

```diff
@@ -46,62 +49,51 @@ var CredentialsProcess = cli.Command{
 			log.Fatal("It looks like you haven't setup your GrantedApprovalsUrl\nTo do so please run: " + c.App.Name + " setup")
 		}
 
-		if profileName == "" {
-			log.Fatalln("Mis-configured aws config file.\n--profile flag must be passed")
-		}
-
-		var profile *cfaws.Profile
-
 		profiles, err := cfaws.LoadProfiles()
 		if err != nil {
-			return err
+			log.Fatal(err)
```

Comment:
Is there any specific need here to use log.Fatal? If we return `err`, `urfav/cli` should log the error for us!

```suggestion
			return err
```